test_pull_requests

```diff
@@ -152,6 +178,97 @@ def submitted_tests_for_branch(branch)
   $submitted_tests[branch] ? $submitted_tests[branch] : $submitted_tests['*']
 end
 
+# Pulls out as much metadata from the bot comment as it can, including
```

Comment:
I've been using Golang comment syntax, unclear if that's the best choice but as far as consistency goes...
```rb
# extract_bot_comment_fields pulls out as much metadata...
```